kv/dist_sender.go

```diff
@@ -252,6 +252,11 @@ func (ds *DistSender) verifyPermissions(args proto.Request) error {
 		})
 }
 
+// lookupOptions capture additional options to pass to InternalRangeLookup.
+type lookupOptions struct {
```

Comment:
Do you anticipate other options to be added to this struct in the future? Although I actually kind of like the struct even if it's a single bool; it makes the code more self-documenting than a plain true/false.